src/storage/store/list_chunk_data.cpp

```diff
@@ -112,9 +116,13 @@ void ListChunkData::resetToEmpty() {
     ColumnChunkData::resetToEmpty();
     sizeColumnChunk->resetToEmpty();
     offsetColumnChunk->resetToEmpty();
-    dataColumnChunk =
-        ColumnChunkFactory::createColumnChunkData(ListType::getChildType(this->dataType).copy(),
-            enableCompression, 0 /* capacity */, ResidencyState::IN_MEMORY);
+    // TODO(bmwinger): can this just be made resetToEmpty?
```

Comment:
I think so.

Reply:
I initially wasn't sure as it was added in #2706 and I thought there might have been some subtle issue with resetToEmpty that caused a bug.
But actually the behaviour replaced in #2706 was calling `unique_ptr::reset` (where the default resets to nullptr), not `resetToEmpty`; so this is probably fine.